### DISCUSSION TITLE
What is the redo-bookmarklet-system branch?

### DISCUSSION THREAD

**Discussion by Shluffy** [Scratch-Bookmarklets/Scratch-Bookmarklets.github.io#124] [Q&A] [answered]
^

**Comment by Shluffy** [comment]
@xItzDiamondx @FunctionalMetatable @gosoccerboy5 what is it?

**Reply by gosoccerboy5** [reply]
idek

**Comment by FunctionalMetatable** [comment, marked as answer]
ahh I'm working on making it a bit better (i think I should've started developing in my fork)

**Reply by Shluffy** [reply]
it's behind on commits. should i bother keeping it up to date

**Reply by FunctionalMetatable** [reply]
@Shluffy nah

**Reply by FunctionalMetatable** [reply]
I'll be moving it to my own branch soon anyways
